File: JKW/JW/os.java
import java.io.File;
import java.io.IOException;

public class jw_sys throws IOExpection {
	public static void create_directory(String name_directory_create) { File dir = new File(name_directory_create); }
	public static void remove_directory(String name_directory_remove) { Files.delete(name_directory_remove); }
	public static void ext(int status) { exit(status); }
	public static void ret(boolean boolean_return) { return boolean_return; }
}

File: JKW/KW/os.kt
class kw_os {
	fun ext(Int status) { exit(status) }
	fun ret(Boolean return_status) { return return_status }
	fun create_directory(String name_directory_create) { File.mkdir(name_directory_create); }
	fun remove_directory(String name_directory_remove) { File.rmdir(name_directory_remove); }
	fun systemd(name_command: String):Int { Runtime.getRuntime().exec(name_command); }
}
